js/app.js

```diff
@@ -101,14 +101,30 @@ function buildBlogSummary(blog, modularContent)
         }
     }
 
-    const body = firstOrDefaultValue(blog, "body") || "";
+    let body = firstOrDefaultValue(blog, "body") || "";
+    body = convertImagesToHyperlink(body);
 	blogSummary.querySelector(".summary-body").innerHTML = body;
 
     processContentSnippets(blogSummary, modularContent);
 	return blogSummary;
 }
 
 
+function convertImagesToHyperlink(blogBody) {
+    let wrapper = document.createElement("div");
+    wrapper.innerHTML = blogBody;
+    wrapper.querySelectorAll("img").forEach(image => {
+        let elementHyperlink = document.createElement('a');
```

Comment:
you could use a template for this i suppose fwiw

Reply:
It may keep it consistent... I'll think about it, for another time